src/name/pubsub/subs.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+const promisify = require('promisify-es6')
+
+const transform = function (res, callback) {
+  callback(null, {
+    strings: res.Strings
+  })
```

Comment:
Please can we just return an array here (and detect if go-ipfs returns `null` and change it to an empty array)